test/phpunit.xml

```diff
@@ -13,6 +13,7 @@
         </testsuite>
 
         <testsuite name='LorisModuleIntegrationTests'>
+            <directory>../raisinbread/test/api/</directory>
```

Comment:
@driusan @kongtiaowang Should we create a new test suite for raisinbread?

Reply:
makes sense to make one to me, but I don't mind if it's just under IntegrationTests for this PR..